package.json

```diff
@@ -2,6 +2,7 @@
   "name": "tota11y",
   "version": "0.1.6",
   "description": "An accessibility visualization toolkit",
+  "main": "build/tota11y.min.js",
```

Comment:
This should be the non-minified version.

Reply:
I am pretty sure that I was getting some errors when using the non-minified version, but let me check that again.

Reply:
Thanks! Really appreciate you donating some of your time to this project.

Reply:
Yep, [tota11y.js#L10137](https://github.com/Khan/tota11y/blob/master/build/tota11y.js#L10137):

```javascript
var newContent = require("!!../node_modules/css-loader/index.js!../node_modules/postcss-loader/index.js!../node_modules/less-loader/index.js!./tota11y.less")
```

…causes the following error:

```
Error: Cannot find module '!!./../node_modules/css-loader/index.js!./../node_modules/postcss-loader/index.js!./../node_modules/autoprefixer-loader/index.js?{browsers:['> 1%']}!./../node_modules/less-loader/index.js!./tota11y.less' from '<…>/node_modules/tota11y/build'
```

I guess that someone forgot to build tota11y in _production mode_ because it should get optimized by [Webpack#L39](https://github.com/Khan/tota11y/blob/master/webpack.config.babel.js#L39).

Reply:
Good find! I'll be doing a new release soon, so I'll make sure to do things right so that this works.

Reply:
So, using `tota11y.js` should work, but it doesn't due to a bad release.

Reply:
I was confused by webpacks development mode when working on another project.  I think development is intended strictly for development.  If we want to ship both `tota11y.min.js` and `tota11y.js` we'll want to use production mode for both, but then toggle minification using a different environment variable or something.

Reply:
@Bartozzz Given that we need to address the build issue before we can use `tota11y.js`, I think we should get this in.

This should also address #85, right?

Reply:
Yes, it should be fixed now.

Reply:
Thanks again for the help on this project and your patience.

Onward!